wpilib/wpilib/counter.py

```diff
@@ -207,16 +207,12 @@ def setUpSource(self, *args, **kwargs):
         and a keyword argument for the same channel.
 
         :param channel: the DIO channel to use as the up source. 0-9 are on-board, 10-25 are on the MXP
-        :type channel: int
         :param source: The digital source to count
-        :type source: DigitalInput
         :param analogTrigger:
             The analog trigger object that is used for the Up Source
-        :type analogTrigger: AnalogTrigger
         :param triggerType:
             The analog trigger output that will trigger the counter.
             Defaults to kState if not specified.
-        :type triggerType: AnalogTrigger.AnalogTriggerType
```

Comment:
Huh? Why did you delete these?

Reply:
Same reason I deleted the return types?

Reply:
Then where are the corresponding type hints?

Reply:
hehe classic find/replace at it again. I forgot about the vararg functions...\*sigh\*

Reply:
I think you need to utilise find and replace better…